docs/README.rst

```diff
@@ -126,12 +126,17 @@ Undo the effects of ``salt.pkgrepo`` on Debian, RedHat, and SuSE.
 
 Clone selected `Salt formulas
 <http://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_
-Git repositories under ``/srv/formulas`` and makes them available in the
-relevant ``file_roots`` settings. Pillar data can be used to customize all
-paths, URLs, etc.
+Git repositories under ``/srv/formulas`` and makes them available in the relevant ``file_roots`` settings. Please note that in order for `file_roots` to be updated, `salt.master` must be called after `salt.formulas`. For example:
```

Comment:
Need to use double-backticks in reStructuredText:

```suggestion
Git repositories under ``/srv/formulas`` and makes them available in the relevant ``file_roots`` settings. Please note that in order for ``file_roots`` to be updated, ``salt.master`` must be called after ``salt.formulas``. For example:
```

@dafyddj I'm sure one of our linters was supposed to catch this but that doesn't seem to be the case.  Any ideas?